libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -669,15 +662,15 @@ uint32_t AP_Frsky_Telem::calc_batt(uint8_t instance)
 {
     const AP_BattMonitor &_battery = AP::battery();
 
-    uint32_t batt;
+    uint32_t batt = 0;
```

Comment:
Move the definition of this down to where it's first assigned to?
```suggestion
```

Reply:
not my code but I agree :-)